### ISSUE TITLE
Doc sidebar should expand for sub-sections like "Other"

### ISSUE THREAD

**Issue by austinkelleher** [marko-js/markojs-website#13]
When on the "Other" section under "Server Integrations," the sub-sections should be expanded in the doc-sidebar:

![screen shot 2017-05-11 at 10 17 25 am](https://cloud.githubusercontent.com/assets/3771924/25954074/2e96327a-3633-11e7-8e6e-e5860904770a.png)

In this case, "Hucwot" should be shown under the "Other" section in the doc-sidebar.

**Comment by newyork-anthonyng** [comment]
Anyone working on this issue? 

I'd like to pick up this ticket if not.

**Comment by austinkelleher** [comment]
Nobody is currently working on this. Feel free to take it on! Let me know if you have any questions.

**Comment by newyork-anthonyng** [comment]
Awesome. I'll try to make a PR this weekend.

**Comment by newyork-anthonyng** [comment]
I ran into this error below when running `npm run start`.

```shell
[marko-starter plugins] Installed plugin: lasso
[marko-starter plugins] Installed plugin: generic-http-server
[marko-starter init] Error building project. Error: A route must contain a `route.js` file that exports a template or handler, or the route must contain an `index.marko` file.
At: /Users/antng/Desktop/Projects/markojs-website
    at Object.onRoute (/Users/antng/Desktop/Projects/markojs-website/node_modules/marko-starter/src/util/loadFilesystemRoutes.js:32:17)
    at buildRoute (/Users/antng/Desktop/Projects/markojs-website/node_modules/routes-table/index.js:122:25)
    at addRoutes (/Users/antng/Desktop/Projects/markojs-website/node_modules/routes-table/index.js:73:42)
    at build (/Users/antng/Desktop/Projects/markojs-website/node_modules/routes-table/index.js:46:9)
    at Promise (/Users/antng/Desktop/Projects/markojs-website/node_modules/routes-table/index.js:25:9)
    at Object.exports.build (/Users/antng/Desktop/Projects/markojs-website/node_modules/routes-table/index.js:24:19)
    at module.exports (/Users/antng/Desktop/Projects/markojs-website/node_modules/marko-starter/src/util/loadFilesystemRoutes.js:12:22)
    at _triggerProjectHook.then (/Users/antng/Desktop/Projects/markojs-website/node_modules/marko-starter/src/util/createProject.js:86:14)
    at process._tickCallback (internal/process/next_tick.js:103:7)
```

This worked for me last week. But a fresh install of the repository produced this error.
@austinkelleher Any ideas about why this is happening?

**Comment by austinkelleher** [comment]
@newyork-anthonyng Related to #16. Investigating.

**Comment by austinkelleher** [comment]
@newyork-anthonyng Fixed #16
